src/Server/PostgreSQLHandler.cpp

```diff
@@ -286,7 +286,8 @@ void PostgreSQLHandler::processQuery()
             settings.max_query_size,
             settings.max_parser_depth,
             settings.max_parser_backtracks,
-            settings.allow_settings_after_format_in_insert);
+            settings.allow_settings_after_format_in_insert,
+            settings.implicit_select);
```

Comment:
Why would somebody use it with PostgreSQL protocol? Maybe pass `false` here?

Reply:
Currently, the PostgreSQL protocol serves as an interface that accepts the same queries as the ClickHouse protocol, using the same syntax. That's why we apply the same rule here.